docs/en/basic/Command-Line-Interface.md

```diff
@@ -26,6 +26,11 @@ Usage: alluxio [COMMAND]
 This section lists usages and examples of general Alluxio operations with the exception of file
 system commands which are covered in the [Admin CLI doc]({{ '/en/operation/Admin-CLI.html' | relativize_url }}).
 
+### extensions
+
+The `extensions` command is meant to for managing any extensions to Alluxio. For additional information, refer
```

Comment:
`any` -> `UFS`. Now `extensions` only used to install UFS extensions.

Reply:
done